src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -98,4 +116,12 @@ private String getUserId(HttpServletRequest request) throws IOException {
 
     return userId;
   }
+
+  private Entity loadRoom(String roomId) {
```

Comment:
something to think about for production code is the non-happy path. 

imagine that an invalid room id was passed in here. What would happen?

p.s. I wouldn't actually fix this in this PR. Just something to remember and consider _and_ test for future